apps/testfront/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Comment regarding all `testfront` files

---

What about creating a small script (node/bash) to check that `storefront` and `testfront` are syncronized in common files? 

Because it might happen that they are a bit different (e.g. someone changed the file in SF but forget/didn't know about TF) and it will cause some issues (e2e tests might work, but UI can be different, etc.)

Or another option is to just copy common files from `SF` to `TF` before each build (with some kind of plugin or something?)

Reply:
https://spryker.atlassian.net/browse/HRZ-2693